<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wContentSearch</class>
 <widget class="QMainWindow" name="wContentSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wContentSearch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources.qrc">
    <normaloff>:/rec/data/icons/iconSearch.svg</normaloff>:/rec/data/icons/iconSearch.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="gbxSearchInput">
       <property name="title">
        <string>Search input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblEnteredFiles_2">
            <property name="text">
             <string>Search input:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddList">
            <property name="text">
             <string>Add files...</string>
            </property>
            <property name="icon">
             <iconset resource="../../ressources.qrc">
              <normaloff>:/rec/data/icons/iconList.svg</normaloff>:/rec/data/icons/iconList.svg</iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../ressources.qrc">
              <normaloff>:/rec/data/icons/iconOpen.svg</normaloff>:/rec/data/icons/iconOpen.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lwgUserSearch">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="childrenCollapsible">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="btnClearLists">
                <property name="text">
                 <string>Clear lists</string>
                </property>
                <property name="icon">
                 <iconset resource="../../ressources.qrc">
                  <normaloff>:/rec/data/icons/iconBroom.svg</normaloff>:/rec/data/icons/iconBroom.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnRemove">
                <property name="text">
                 <string>Remove selection</string>
                </property>
                <property name="icon">
                 <iconset resource="../../ressources.qrc">
                  <normaloff>:/rec/data/icons/iconDelete.svg</normaloff>:/rec/data/icons/iconDelete.svg</iconset>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="pgbProgress">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnSearch">
                <property name="font">
                 <font>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Search!</string>
                </property>
                <property name="icon">
                 <iconset resource="../../ressources.qrc">
                  <normaloff>:/rec/data/icons/iconSearch.svg</normaloff>:/rec/data/icons/iconSearch.svg</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="gbxResults">
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="twgExtras">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabLinks">
           <attribute name="title">
            <string notr="true">tabLinks</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QListWidget" name="lwgLinks">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblLinkInfo">
              <property name="text">
               <string extracomment="Use plural form for 'Information'">Information for selected link:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="pteInformation">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="btnOpenInBrowser">
                <property name="text">
                 <string>Open in browser...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../ressources.qrc">
                  <normaloff>:/rec/data/icons/iconWorld.svg</normaloff>:/rec/data/icons/iconWorld.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnCopy">
                <property name="text">
                 <string>Copy</string>
                </property>
                <property name="icon">
                 <iconset resource="../../ressources.qrc">
                  <normaloff>:/rec/data/icons/iconCopy.svg</normaloff>:/rec/data/icons/iconCopy.svg</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btnReportDeathLink">
              <property name="toolTip">
               <string>Press this button if a link provided by the program no longer works or does not contain a file.</string>
              </property>
              <property name="text">
               <string>Report invalid link...</string>
              </property>
              <property name="icon">
               <iconset resource="../../ressources.qrc">
                <normaloff>:/rec/data/icons/iconDelete.svg</normaloff>:/rec/data/icons/iconDelete.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabNotFound">
           <attribute name="title">
            <string notr="true">tabNotFound</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QListWidget" name="lwgNotFound">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabStandardContent">
           <attribute name="title">
            <string notr="true">tabStandardContent</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QListWidget" name="lwgStandardContent"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabDirectLinks">
           <attribute name="title">
            <string notr="true">tabDirectLinks</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QListWidget" name="lwgDirectLinks"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionAddList"/>
    <addaction name="actionRemoveSelection"/>
    <addaction name="separator"/>
    <addaction name="actionBackToHome"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSpecials">
    <property name="title">
     <string>Specials</string>
    </property>
    <addaction name="actionSuggestFiles"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionAddExamples"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSendFeedback"/>
   </widget>
   <addaction name="menuSearch"/>
   <addaction name="menuSpecials"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconSearch.svg</normaloff>:/rec/data/icons/iconSearch.svg</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconQuit.svg</normaloff>:/rec/data/icons/iconQuit.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconPreferences.svg</normaloff>:/rec/data/icons/iconPreferences.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
  </action>
  <action name="actionRemoveSelection">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconDelete.svg</normaloff>:/rec/data/icons/iconDelete.svg</iconset>
   </property>
   <property name="text">
    <string>Remove selection</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionAddList">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconList.svg</normaloff>:/rec/data/icons/iconList.svg</iconset>
   </property>
   <property name="text">
    <string>Add list...</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconBug.svg</normaloff>:/rec/data/icons/iconBug.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string notr="true">F1</string>
   </property>
  </action>
  <action name="actionSuggestFiles">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconLight.svg</normaloff>:/rec/data/icons/iconLight.svg</iconset>
   </property>
   <property name="text">
    <string>Suggest files...</string>
   </property>
  </action>
  <action name="actionBackToHome">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconHome.svg</normaloff>:/rec/data/icons/iconHome.svg</iconset>
   </property>
   <property name="text">
    <string>Back to home</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="actionAddExamples">
   <property name="text">
    <string notr="true">Add examples</string>
   </property>
   <property name="iconText">
    <string notr="true">Add examples</string>
   </property>
   <property name="toolTip">
    <string notr="true">Add examples</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
